ui/perfherder/graphs/TestDataModal.jsx

```diff
@@ -391,8 +400,11 @@ export default class TestDataModal extends React.Component {
         title: 'Framework',
       },
       {
-        options: projects.length ? projects.map(item => item.name) : [],
+        options: this.getDropdownOptions(projects),
```

Comment:
The result of `this.getDropdownOptions(projects)` can be extracted into a new local variable.
No need for computing again the dropdown options based on provided projects.